client/blocks/plugins-update-manager/index.tsx

```diff
@@ -91,8 +93,8 @@ export const PluginsUpdateManager = ( props: Props ) => {
 			<MainComponent wideLayout>
 				<NavigationHeader
 					navigationItems={ [] }
-					title="Plugin updates manager"
-					subtitle="Effortlessly schedule plugin auto-updates with built-in rollback logic."
+					title={ translate( 'Plugins update scheduler' ) }
```

Comment:
```suggestion
					title={ translate( 'Scheduled Updates' ) }
```

Reply:
Would be great to keep "plugin" in the title, since that's the most prominent text on the page.  I'd propose a slight tweak to the original: **Plugin Update Manager**

This will also cut down on the number of times the word `schedule` appears on the page—four (title, subtitle, CTA, and table title) could be overkill.

Reply:
Thanks! I updated the suggestion. 

I've tried to keep away from the word "Manager" because I don't think it's super substantive, but I don't feel strongly enough to keep this from moving forward :)

Reply:
`Plugin Update Center` ?